docs/resource-schemas.md

```diff
@@ -53,6 +53,20 @@ assets:
       content.
 ```
 
+## Stable schema version
+
+The current supported stable version of the resource schemas is _v1.1.5_. You can add the yaml
+language server to your YAML files by adding the following line to the top of your files:
+
+```yml
+# yaml-language-server: $schema=https://unpkg.com/@carbon-platform/schemas@1.1.5/carbon-resources.schema.json
+```
```

Comment:
I'd recommend that we only specify support on the major version level and ensure we follow semantic versioning for this package.

Reply:
Done!